app/modals.R

```diff
@@ -61,15 +61,15 @@ cromwellStartModal <- function(failed = FALSE, error = "An error occurred") {
   )
 }
 
-verifyCromwellDeleteModal <- function(failed = FALSE, error = "Woops, an error! Contact DaSL") {
+verifyCromwellDeleteModal <- function(failed = FALSE, error = "Woops, an error! Contact DaSL at wilds@fredhutch.org.") {
   modalDialog(
-    title = "Delete your PROOF Cromwell server",
-    "Permanently delete your PROOF Cromwell server. Although you can't undo this action, you can start up another one anytime!",
+    title = "Stop your PROOF server",
+    "Permanently stop your PROOF server. Although you can't undo this action, you can start up another one anytime!",
```

Comment:
We should remove the word "Permanently" - makes it sound like they can't get another one

Reply:
Yeah, you're right, I was trying to soft it by removing delete, but now it does look a little crazy to say "permanently". haha

Reply:
okay, cool